runtime/syntax/obl.vim

```diff
@@ -0,0 +1,3354 @@
+" Vim syntax file
+" Language: Oblivion Language (obl)
+" Original Creator: Ulthar Seramis
+" Maintainer: Kat <katisntgood@gmail.com>
+" Latest Revision: 13 November 2022
+
```

Comment:
Could you please set and restore `'cpo'` appropriately to handle the line continuations in this file?

See `:help line-continuation` for details and https://github.com/search?q=repo%3Avim%2Fvim%20cpo_save&type=code for examples.